content/advanced_usage/compare_pipelines.md

```diff
@@ -43,3 +43,22 @@ Steps to select a particular instance from the history:
 1. Click on the search box
 2. Click on "Browse the timeline"
 3. Browse the history and select a pipeline
+
+## Understanding Non-sequential Material Revision
+
+Sometimes the following error message will be present on the comparison page:
+
+```text
+This pipeline instance was triggered with a non-sequential material revision.
+```
+
+This means that the instance in-question was triggered with an older revision or an older run of the upstream pipeline. This occurs when the `Trigger with options` flow is chosen on the Dashboard UI to schedule a pipeline.
+
+Let's consider a pipeline `P` with material `M`.
+
+ - a commit `c1` trigger an instance `P1` with a `natural_order` of 1
+ - a commit `c2` trigger an instance `P2` with a `natural_order` of 2
+ - a commit `c3` trigger an instance `P3` with a `natural_order` of 3
+
+Now a user using trigger with options triggers an instance `P4` with commit `c3`. Now, in all manners `P2` and `P4` are similar.
```

Comment:
When you say `c3` in this line and the next, don't you mean `c2`?

Reply:
Yes I do. Will update it.